compiler/noirc_frontend/src/parser/parser/expression.rs

```diff
@@ -128,12 +159,11 @@ impl Parser<'_> {
         Some(atom)
     }
 
-    /// UnaryRightExpression
+    /// MemberAccessOrCallOrMethodCallOrIndexExpression
     ///     = CallExpression
     ///     | MemberAccessOrMethodCallExpression
-    ///     | CastExpression
     ///     | IndexExpression
-    fn parse_unary_right(
+    fn parse_member_access_or_call_or_method_call_or_index(
```

Comment:
Let's rename this one too.

This can be `atom_rhs` or `high_prec_postfix_op`, etc depending on the name you choose for my previous comment

Reply:
In the end I went with `atom_rhs`